src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -225,14 +230,6 @@ public class RunMojo extends AbstractJettyMojo {
     @Parameter
     protected boolean pluginFirstClassLoader = false;
 
-    /**
-     * List of additional System properties to set
-     *
-     * @since 1.85
-     */
-    @Parameter
-    private Map<String, String> systemProperties;
-
```

Comment:
This is now present in the superclass so there is no need to maintain our own shadow field.